Source/Core/VideoCommon/LightingShaderGen.h

```diff
@@ -149,8 +149,8 @@ static void GenerateLightingShader(T& object, LightingUidData& uid_data, int com
 				else
 					// TODO: this isn't verified. Here we want to read the ambient from the vertex,
 					// but the vertex itself has no color. So we don't know which value to read.
-					// Returing 1.0 is the same as disabled lightning, so this could be fine
-					object.Write("lacc = int4(255, 255, 255, 255);\n");
+					// Those boxes in SMS seems to need 0.
```

Comment:
"Those boxes in SMS" is vague. Maybe you could write something like "Boxes in Super Mario Sunshine bonus stages seem to incorrectly be visible if this isn't 0".